cmd/jaeger/internal/integration/grpc_test.go

```diff
@@ -5,12 +5,47 @@ package integration
 
 import (
 	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
 )
 
+type GRPCStorageIntegration struct {
+	E2EStorageIntegration
+
+	server *integration.GRPCServer
+}
+
+func (s *GRPCStorageIntegration) initialize(t *testing.T) {
+	var err error
+	s.server, err = integration.NewGRPCServer()
+	require.NoError(t, err)
+	require.NoError(t, s.server.Start())
+
+	s.Refresh = func(_ *testing.T) {}
+	s.CleanUp = s.cleanUp
+}
+
+func (s *GRPCStorageIntegration) cleanUp(t *testing.T) {
+	require.NoError(t, s.server.Close())
```

Comment:
I am curious, is it really enough to close the server, which only closes the connection to remote process, but doesn't actually perform storage clean-up? The intention of CleanUp was to delete data from storage to avoid data interference across tests.